src/main/java/nl/dtls/fairdatapoint/config/RepositoryConfig.java

```diff
@@ -52,7 +52,7 @@ public class RepositoryConfig {
     @Autowired
     private RepositoryProperties repositoryProperties;
 
-    @Bean(initMethod = "initialize", destroyMethod = "shutDown")
+    @Bean(initMethod = "init", destroyMethod = "shutDown")
```

Comment:
Are these all the changes needed for the 4.x upgrade? 😮

Reply:
Yes 😄 https://rdf4j.org/release-notes/4.0.0-m3/

> The deprecated initialize() methods on Repository, Sail and RepositoryManager have been removed. Use init() instead. (see [GH-1243](https://github.com/eclipse/rdf4j/issues/1243))

Reply:
Luckily almost none of the changes in API affect us...

Reply:
@kburger anything else should be solved here or we can merge?

Reply:
Seems to be OK, I'll try to read up on the release notes and see if there are any others changes that we might benefit from. For now we can merge this.